test/test_dataset.py

```diff
@@ -44,9 +43,9 @@ def test_open_single_dataset(self):
 
         uds3 = ux.open_dataset(uds3_name, uds3_data_name1)
 
-        n3 = uds3.ds_var_names["Mesh2_node_x"]
+        n3 = uds3.Mesh2_node_x
```

Comment:
I think the name "n3" was also referring to the fact that this was the variable name, but it is the variable itself from now on with this change. So, naming it as "n3" won't make sense. The same comment above could apply to this one; no need for such a variable definition, and `uds3.Mesh2_node_x` could be directly used below I think.